openssl.go

```diff
@@ -267,20 +243,22 @@ func baseNeverEmpty(b []byte) *C.uchar {
 	return (*C.uchar)(unsafe.Pointer(&b[0]))
 }
 
-// base returns the address of the underlying array in b,
-// being careful not to panic when b has zero length.
-func base(b []byte) *C.uchar {
+// pbase returns the address of the underlying array in b.
```

Comment:
Should this also have the carefulness comment? If it doesn't, it seems like by contrast `pbase` *won't* be careful while `base` would be careful.

```suggestion
// pbase returns the address of the underlying array in b,
// being careful not to panic when b has zero length.
```

While here, maybe add a comment to `sbase` too?